### DISCUSSION TITLE
Recording both internal and external audio (LTC Timecode)

### DISCUSSION THREAD

**Discussion by WorkingTensPhilip** [gopro/labs#486] [General]
I am trying to inject LTC timecode via external source while also recording a scratch track from the internal mics, is this possible?

**Comment by dnewman-gpsw** [comment]
Moved to discussions.  The camera will encode stereo, but only from a single source external or internal.  So inject scratch audio with LTC for this, some devices like the Tenacle Sync gear have microphone to do this automatically.
